lib/jxl/fast_math_test.cc

```diff
@@ -107,6 +107,26 @@ HWY_NOINLINE void TestFastErf() {
   printf("max abs err %e\n", static_cast<double>(max_abs_err));
 }
 
+HWY_NOINLINE void MaxCubeRootError() {
+  const HWY_FULL(float) d;
+  float max_err = 0.0f;
+  for (uint64_t x5 = 0; x5 < 2000000; x5++) {
+    const float x = x5 * 1E-5f;
+    const float expected = cbrtf(x);
+    HWY_ALIGN float approx[MaxLanes(d)];
+    Store(CubeRootAndAdd(Set(d, x), Zero(d)), d, approx);
+
+    // All lanes are same
+    for (size_t i = 1; i < Lanes(d); ++i) {
+      EXPECT_LE(std::abs(approx[0] - approx[i]), 5E-7f);
```

Comment:
https://google.github.io/googletest/reference/assertions.html#EXPECT_NEAR should give better error messages.

Can we see what failures we get with 0 here?

Reply:
Let's take a look...

Reply:
So, the result are errors like this:

```
The difference between approx[0] and approx[i] is 2.384185791015625e-07, which exceeds 0, where
approx[0] evaluates to 2.635331392288208,
approx[i] evaluates to 2.6353311538696289, and
0 evaluates to 0.
```
I will set the tolerance to 5E-7f again, for greener tests.